vivyjava/src/main/resources/META-INF/plugin.xml

```diff
@@ -11,16 +11,18 @@
 
     <extensions defaultExtensionNs="com.intellij">
         <!-- Add your extensions here -->
+        <projectConfigurable instance="settings.SettingsPage" />
     </extensions>
 
     <depends>com.intellij.modules.lang</depends>
-    <depends>CheckStyle-IDEA</depends>
+    <!--<depends>CheckStyle-IDEA</depends>-->
```

Comment:
why turned off?

Reply:
restored.. off during debugging as it was not available in the debug instance